libs/common-utils/src/fractionUtils.test.ts

```diff
@@ -42,4 +44,37 @@ describe('Fraction utils', () => {
       expect(JSBI.toNumber(fraction.denominator)).toBe(100000)
     })
   })
+
+  describe('simplify', () => {
```

Comment:
is the base branch correct? was this intended change in this PR?

Reply:
Yes, I mentioned it in the PR description.

Will clean it up